src/cmd/terraform.go

```diff
@@ -310,30 +310,49 @@ func exportServices(c *opslevel.Client, shell *os.File, directory string) {
 func exportTeams(c *opslevel.Client, config *os.File, shell *os.File) {
 	shell.WriteString("# Teams\n")
 
-	teamConfig := `resource "opslevel_team" "%s" {
-  name = "%s"
-  manager_email = "%s"
-  parent = "%s"
-  %s
-  %s
+	teamConfig := `resource "opslevel_team" "%s" {%s
 }
+
 `
 	resp, err := c.ListTeams(nil)
 	teams := resp.Nodes
 	cobra.CheckErr(err)
 	for _, team := range teams {
+		teamBody := ""
+
 		aliases := flattenAliases(team.Aliases)
 		if len(aliases) > 0 {
-			aliases = fmt.Sprintf("aliases = [\"%s\"]", aliases)
+			teamBody += fmt.Sprintf("\n  aliases = [\"%s\"]", aliases)
 		}
+		teamBody += fmt.Sprintf("\n  name = \"%s\"", team.Name)
+
+		if len(team.Group.Alias) > 0 {
+			teamBody += fmt.Sprintf("\n  group = \"%s\"", team.Group.Alias)
+		}
+		membersOutput := ""
+		for _, member := range team.Memberships.Nodes {
+			memberConfig := `
+  member {
+    email = "%s"
+    role = "%s"
+  }`
+			membersOutput += fmt.Sprintf(memberConfig, member.User.Email, member.Role)
+		}
+		if len(membersOutput) > 0 {
```

Comment:
This `if len()` check is safe to remove since appending an empty string doesn't really change teamBody

Reply:
Actually where `membersOutput` starts down to here seems like a good candidate for a small function. Lmk if you would like to discuss :) 
Something loosely like `membersOutput := getMembershipsAsTerraformConfig(team.Memberships.Nodes)`

Reply:
Implemented these suggestions